### PR TITLE
Python 3 compatibility

### PR DESCRIPTION
**Pull request by gaetano-guerriero**

### CODE REVIEW

File: test/test_debianbts.py [hunk 1]
```diff
@@ -111,9 +115,9 @@ def testGetBugLog(self):
             self.assertTrue("attachments" in i)
             self.assertEqual(type(i["attachments"]), type(list()))
             self.assertTrue("body" in i)
-            self.assertEqual(type(i["body"]), type(unicode()))
+            self.assertTrue(isinstance(i["body"], basestring))
```
Comment (venthur) [inline test/test_debianbts.py]: This evaluates to `True` in Python2 for strings and unicode strings.

File: test/test_debianbts.py [hunk 2]
```diff
@@ -111,9 +115,9 @@ def testGetBugLog(self):
...
             self.assertTrue("header" in i)
-            self.assertEqual(type(i["header"]), type(unicode()))
+            self.assertTrue(isinstance(i["header"], basestring))
```
Comment (venthur) [inline test/test_debianbts.py]: This evaluates to `True` in Python2 for strings and unicode strings.

File: src/debianbts.py [hunk 3]
```diff
@@ -52,6 +54,7 @@
 soap_client = SoapClient(location=URL, namespace=NS, soap_ns='soap')
 
 
+@functools.total_ordering
```
Comment (venthur) [inline src/debianbts.py]: I know I'm a PITA, but could you please rewrite it w/o using total_ordering by providing the whole set of comparison methods?

**Comment by venthur** [comment]
Wow, that was rather quick! I hope I have some time tomorrow to have an in-depth review, otherwise we'll have to wait two weeks.

Thank you again for your patch, you're doing great work!

File: src/debianbts.py (hunk 3 above)
Comment (gaetano-guerriero) [inline src/debianbts.py]: Certainly, can I ask you why out of curiosity ?

File: test/test_debianbts.py (hunk 1 above)
Comment (gaetano-guerriero) [inline test/test_debianbts.py]: ok fixed !

**Comment by venthur** [comment]
Awesome stuff! I've added a few minor things (several future imports, all strings are unicode by default), but everything works on python2 and 3 now.

Thank you very much!

**Pull request merged**